src/common/NetworkManager.ts

```diff
@@ -258,11 +253,7 @@ export class NetworkManager extends EventEmitter {
   }
 
   _cacheDisabled(): boolean {
-    return (
-      this._userCacheDisabled ||
-      (this._userRequestInterceptionEnabled &&
-        !this._userRequestInterceptionCacheSafe)
-    );
+    return this._userCacheDisabled;
```

Comment:
This is the main change of the PR.

Reply:
People expect interception to be independent of caching and that's how it was before #1154.